src/main/java/org/jenkinsci/plugins/credentialsbinding/impl/BindingStep.java

```diff
@@ -181,10 +182,15 @@ private void doStart() throws Exception {
 
         @Override public void expand(EnvVars env) throws IOException, InterruptedException {
             for (Map.Entry<String,Secret> override : overrides.entrySet()) {
-                env.override(override.getKey(), override.getValue().getPlainText());
+                String keyOverride = override.getKey();
+                env.override(keyOverride, override.getValue().getPlainText());
+                sensitiveVariables.add(keyOverride);
```

Comment:
```suggestion
                env.override(override.getKey(), override.getValue().getPlainText());
```

Reply:
addressed in 06428a0516322ded27dffb274fdcf04017ab944e